project.clj

```diff
@@ -24,7 +24,7 @@
                  [org.cyverse/authy "2.8.0"]
                  [org.cyverse/clojure-commons "2.8.0"]
                  [org.cyverse/kameleon "2.8.0"]
-                 [org.cyverse/mescal "2.8.0"]
+                 [org.cyverse/mescal "2.8.1-SNAPSHOT"]
```

Comment:
Should this `SNAPSHOT` dependency be committed?
Or should this just be `2.8.1` after this version of the mescal lib has been released?